week-03/main.js

```diff
@@ -4,6 +4,8 @@
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
 
+list.shift(1);
+list.push(1);
```

Comment:
The push() method adds one or more elements to the end of an array. While this is technically adding `1` to the array, it's not moving the same element that was shifted.